frontend/src/assets/css/material-dashboard.css

```
@@ -21201,12 +21201,8 @@ fieldset:disabled .btn {
   transition: all 300ms cubic-bezier(0.34, 1.61, 0.7, 1);
 }
 
-.card:hover[data-animation="true"] .card-header {
-  -webkit-transform: translate3d(0, -50px, 0);
-  -moz-transform: translate3d(0, -50px, 0);
-  -o-transform: translate3d(0, -50px, 0);
-  -ms-transform: translate3d(0, -50px, 0);
-  transform: translate3d(0, -50px, 0);
+.card:hover[data-animation="false"] .card-header {
+
 }
```

Biome (1.9.4)

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)